### ISSUE TITLE
Feature request: renaming planets

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#801]
```
I think it would be nice to be able to rename planets. there is a lot of 
'numbered' planets, and i think there were a name duplicate in the original 
game (Exterior 19?).

```

Original issue reported on code.google.com by `kli...@gmail.com` on 17 Jan 2014 at 3:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 17 Jan 2014 at 5:53
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**, **Priority-Low**, **Component-UI**
- Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Done in the next release.
```

Original comment by `akarn...@gmail.com` on 21 Feb 2014 at 10:45
- Changed state: **SVN**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.95.163
```

Original comment by `akarn...@gmail.com` on 21 Feb 2014 at 10:59
- Changed state: **Fixed**
